tasks/CentOS/rabbit.yml

```diff
@@ -4,10 +4,32 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Add RabbitMQ's repo
+    yum_repository:
+      name: rabbitmq
+      description: rabbitmq
+      baseurl: "https://dl.bintray.com/rabbitmq/rpm/rabbitmq-server/v3.7.x/el/{{ansible_distribution_major_version}}"
+      gpgcheck: yes
+      gpgkey: https://dl.bintray.com/rabbitmq/Keys/rabbitmq-release-signing-key.asc
```

Comment:
Interesting, Ansible keeps failing to get this in the Travis build:
```
TASK [role_under_test : Ensure the RabbitMQ APT repo GPG key is present] *******
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to download key at https://www.rabbitmq.com/rabbitmq-release-signing-key.asc: Connection failure: ('The read operation timed out',)"}
	to retry, use: --limit @/etc/ansible/roles/role_under_test/tests/test.retry
```

but this seems fine to me, I'll see if I can repro these timeout issues, might just be something on the Travis side.

Reply:
If I'm not mistaken the issue is actually with Ubuntu / Debian roles on Travis isn't it ?

```
TASK [role_under_test : Ensure the RabbitMQ APT repo GPG key is present] *******
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to download key at https://www.rabbitmq.com/rabbitmq-release-signing-key.asc: Connection failure: ('The read operation timed out',)"}
	to retry, use: --limit @/etc/ansible/roles/role_under_test/tests/test.retry
```

This is part of cmacrae.sensu/tasks/Ubuntu/rabbit.yml & cmacrae.sensu/tasks/Debian/rabbit.yml

Reply:
Woooow yeah, I totally missed that. Not related to your change at all. Let me try to unstick them.

Reply:
Working on it, looks like `https://www.rabbitmq.com/rabbitmq-release-signing-key.asc` is being super flakey. Sometime's it pulls instantly, other times, it lags a ton. It also looks like Debian/Ubuntu are using the older rabbitmq mirrors not PackageCloud/Bintray so, that's not ideal either.